build-settings/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "ws-dev-build-settings",
+    "version": "0.0.1",
+    "license": "MIT",
+    "private": true
+}
```

Comment:
Hopefully this `build-settings` internal package pattern continues to be useful. 🤞